integration/js/pages/AppPage.js

```diff
@@ -530,20 +530,9 @@ class AppPage {
     return classes.includes('vf-active');
   }
 
-  // enable voice focus in lobby if not enabled
   async enableVoiceFocusInLobby() {
     const elem = await this.driver.findElement(elements.addVoiceFocusInput);
-    if ( !elem.isSelected() ){
-      return clickElement(this.driver, elem);
-    }
-  }
-
-  // disable voice focus in lobby if enabled
-  async disableVoiceFocusInLobby() {
-    const elem = await this.driver.findElement(elements.addVoiceFocusInput);
-    if ( elem.isSelected() ){
-      return clickElement(this.driver, elem);
-    }
+    return clickElement(this.driver, elem);
```

Comment:
FYI the reason these functions didn't work as expected is elem.isSelected() was not returning whether the 'Add Voice Focus' checkbox was actually checked or not.